### ISSUE TITLE
[Tacotron2-Waveglow/PyTorch] Benchmark with different gpu number

### ISSUE THREAD

**Issue by hadaev8** [NVIDIA/DeepLearningExamples#358]
Related to **Model/Framework(s) or something else (describe)** 
[Tacotron2-Waveglow/PyTorch]

**Is your feature request related to a problem? Please describe.**
Yes, I have a modification of Nvidia implementation of tacotron2.
I noticed loss very unstable and the train perform worse in multi GPU setup.
https://i.imgur.com/ldW0uQw.png

**Describe the solution you'd like**
I would like to compare myself with the official benchmark from Nvidia experts and baseline tacotron.

**Comment by GrzegorzKarchNV** [comment]
Hi @hadaev8 sorry for this very late reply. Were you able to solve this problem? What modifications have you made? You could try decreasing the learning rate for multi-gpu.

**Comment by hadaev8** [comment]
I tried it.
Still, single GPU is better.
A lot of papers use only single gpu train for tacotron (while use multi gpu setup for the vocoder, for example).
So I got the impression that the tacotron is not suitable for multi gpu train.

**Comment by GrzegorzKarchNV** [comment]
We trained our checkpoints using 8GPUs, you can download them here:
https://ngc.nvidia.com/catalog/models/nvidia:tacotron2pyt_fp16/files?version=3

**Comment by GrzegorzKarchNV** [comment]
closing due to inactivity

**Comment by hadaev8** [comment]
@GrzegorzKarchNV 
Well, my question was, whats your final loss on single gpu vs 2-4-8 gpus?
How better train process was.
